src/DataFixtures/AppFixtures.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+namespace App\DataFixtures;
+
+
+
+use App\Entity\ProjectTool;
+use App\Entity\ToolMeasure;
+use DateTimeImmutable;
+use Doctrine\Persistence\ObjectManager;
+use Doctrine\Bundle\FixturesBundle\Fixture;
+
+class AppFixtures extends Fixture
+{
+    public function load(ObjectManager $manager): void
+    {
+        $projectTool = new ProjectTool();
```

Comment:
o gal iseitu sudeti i `for()` ir tada uztektu tik po viena $projectTool ir $toolMeasure ir tiesiog su kiekvienu `for` ciklu pakeistum skaiciu?